Formula/newrelic-infra-agent.rb

```diff
@@ -0,0 +1,40 @@
+class NewrelicInfraAgent < Formula
+  desc "New Relic infrastructure agent"
+  homepage "https://github.com/newrelic/infrastructure-agent"
+  url "https://github.com/newrelic/infrastructure-agent/archive/refs/tags/1.20.0.tar.gz"
+  sha256 "b9f6f744b8376b0ba92657237902d8a8e7c12cbb7f56c6f13f93b52255f31224"
+  license "Apache-2.0"
+  head "https://github.com/newrelic/infrastructure-agent.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["VERSION"] = version
+    ENV["GOOS"] = "darwin"
```

Comment:
I don't think we want this on Linux.

Reply:
The install block below will also fail on linux